package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "lodash": "^4.0.0",
-    "rxjs": "5.0.0-rc.5"
+    "rxjs": "5.4.2"
```

Comment:
The [specific fix needed was in 5.4.2](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#542-2017-07-05), so `~5.4.0` won't do, but I believe `~5.4.2` should work. I'll make the change.